src/views/admin/user/edit-user.hbs

```diff
@@ -51,6 +51,15 @@
                         <input type="text" name="user_email" id="user_email" class="form-control" value="{{user.email}}"
                             placeholder="Type an Email" required />
                     </div>
+                    <div class="form-group">
+                        <label for="user_password"> Password:</label>
+                        <input type="password" name="user_password" id="user_password" class="form-control"
+                            placeholder="Type a Password" min="8" max="20" />
```

Comment:
`placeholder="Type a Password" min="10" max="20" />`

Reply:
I did that change but now I can't save the password, it shows a message "Utiliza un formato que coincida con el solicitado" and I am typing the password with the correct conditions.